src/Controller/ExerciseController.php

```diff
@@ -0,0 +1,95 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Exercise;
+use App\Form\ExerciseType;
+use App\Repository\ExerciseRepository;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Annotation\Route;
+
+/**
+ * @Route("/exercise")
+ */
+class ExerciseController extends BaseController
+{
+    /**
+     * @Route("/", name="exercise_index", methods={"GET"})
+     */
+    public function index(ExerciseRepository $exerciseRepository): Response
+    {
+        return $this->render('exercise/index.html.twig', [
+            'exercises' => $exerciseRepository->findAll(),
```

Comment:
@daria-sieroshtan 
I know this is not final but you may want to add todo for pagination

Reply:
Already in todo.